src/main/java/com/vladmihalcea/sql/exception/SQLStatementCountMismatchAssertionError.java

```diff
@@ -6,12 +6,12 @@
  *
  * @author Vlad Mihalcea
  */
-public abstract class SQLStatementCountMismatchException extends RuntimeException {
+public abstract class SQLStatementCountMismatchAssertionError extends AssertionError {
```

Comment:
Why not just change the `SQLStatementCountMismatchException` extend `AssertionError` instead?